### DISCUSSION TITLE
I can't use Kratos with Pgbouncer.

### DISCUSSION THREAD

**Discussion by meotimdihia** [ory/kratos#3683] [Q&A] [answered]
`DNS="postgresql://user:password@localhost:6432/database"`

While I can do it with Prisma with this connect string: `DATABASE_URL="postgresql://user:password@localhost:6432/database?schema=public&pgbouncer=true"`

Anyone know how to fix this problem.

**Comment by vinckr** [comment, marked as answer]
Hello @meotimdihia 
I am not familiar with pgbouncer, so not really able to help here. 
Why are you required to use pgbouncer? I would suggest to just try it out without if there is no strong requirement for it.
